components/learningUnits/LearningUnitsOfCurriculum.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useOperation } from "react-openapi-client";
+import { Alert, Container } from "@mui/material";
+import Loader from "../common/Loader";
+import LearningUnit from "./LearningUnit";
+
+export default function LearningUnitsOfCurriculum({ curriculumId }) {
+  const {
+    loading: completedLoading,
+    error: completedError,
+    data: completedData,
+  } = useOperation(
+    "listCompletedLearningUnitsOfCurriculumCompletedByUser",
+    curriculumId
+  );
+
+  const {
+    loading: learningUnitsLoading,
+    data: learningUnitsData,
+    error: learningUnitsError,
+  } = useOperation("listLearningUnitsOfCurriculum", curriculumId);
+
+  // const isCompleted = false;
+
+  if (learningUnitsError || completedError) {
+    return <Alert severity="error">Error</Alert>;
```

Comment:
It would be better if the alert displays the error message of the corresponding error (learningUnitsError or completedError)

Reply:
I had that before, but I think in the future we will want to manage the errors ourselves, and not communicating it to the user.
Probably in the near future, creating an error component that shows something cool whenever there is an error, more than retrieving the API error response.

Reply:
Agree, we should find a better way to handle errors and not be displayed to users